### PR TITLE
Task/prepare 2.3.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics PR #609

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,6 +0,0 @@
-- Fix: update tests to not create initial entity when append mode is enabled
-- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iota-node-lib 3.2.0) (iota-node-lib#1384)
-- Fix: update tests according new default value for appendMode
-- Fix: use 'options=upsert' when update ngsiv2 CB entities and appendMode is enabled (iotagent-node-lib#956)
-- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1377)
-- Remove: RPM stuff (no longer used)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:6]: Should be moved to rpm/SPECS/iotaul.spec

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:6]: rpm/SPECS/iotaul.spec was removed

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:6]: Ah ok... the right file is Changelog, in the root of the repository

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:6]: fixed in https://github.com/telefonicaid/iotagent-ul/pull/622/commits/8a7e5870dd3e5e3b470c774c661b5e7db89847ba

File: Changelog [hunk 2]
```diff
@@ -1,3 +1,12 @@
+2.3.0 (Aug 24th, 2023)
+
+- Fix: update tests to not create initial entity when append mode is enabled
+- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iota-node-lib 3.2.0) (iota-node-lib#1384)
+- Fix: update tests according new default value for appendMode
+- Fix: use 'options=upsert' when update ngsiv2 CB entities and appendMode is enabled (iotagent-node-lib#956)
+- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1377)
+- Remove: RPM stuff (no longer used)
```
Comment (fgalan) [inline Changelog:8]:
```suggestion
- Fix: update tests to not create initial entity when append mode is enabled
- Fix: update tests according new default value for appendMode
- Fix: use 'options=upsert' when update ngsiv2 CB entities and appendMode is enabled (iotagent-node-lib#956)
- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1377)
- Upgrade iotagent-node-lib dependency from 3.2.0 to 3.3.0
- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iota-node-lib 3.2.0) (iota-node-lib#1384)
- Remove: RPM stuff (no longer used)
```

File: Changelog [hunk 3]
```diff
@@ -1,3 +1,12 @@
+2.3.0 (Aug 24th, 2023)
```
Comment (fgalan) [inline Changelog:1]:
```suggestion
2.3.0 (Aug 25th, 2023)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**